### ISSUE TITLE
where i can find the file 'cls_labels.npy'

### ISSUE THREAD

**Issue by Chenfeng1271** [JaveyWang/PAN-pytorch#1]
I train the PAN, and download the `classsegmentaug` file and creat `'train_aug.txt'`, however I can't find the `cls_labels.npy`  in `dataset.py`

**Comment by JiaweiWang-AI** [comment]
The file cls_labels.npy saves the class label of each image, format like {'filename' : '1, 0, 1, ..., 0'}. For simplicity, you can use the label from the x_mask, I think the program may seems like 
`y = multi_hot(set(np.array(x_mask).tolist()))
`
In general, the class label is not necessary for semantic segmentation problem, just suit yourself :)

**Comment by John1231983** [comment]
@JaveyWang : How about performance? Is it better than without attention?

**Comment by Chenfeng1271** [comment]
I remove everything in accord with the `class task`, shall I reset the alpha and beta?

**Comment by JiaweiWang-AI** [comment]
> I remove everything in accord with the `class task`, shall I reset the alpha and beta?

No, you shouldn't. The alpha is a classification coefficient. Due to the fact that I have set detach before the classifier, it has no impact on the main networks. You can just delete the relevant part.

**Comment by JiaweiWang-AI** [comment]
> @JaveyWang : How about performance? Is it better than without attention?

Yes. One scale evaluation can get up to 78% mIOU. More details you can see in the PAN paper.

**Comment by Chenfeng1271** [comment]
I reset the image resolution to 512*512, but the whole mIoU is 70% and doesn't change after 40 epochs. Do you know how to refine it as the paper has done?

**Comment by JiaweiWang-AI** [comment]
Any question about the accuracy, please set the batch_size as large as you can, and set the image size to 512 * 512. I train the model in GTX1080ti with 11GB of RAM before.
